sass/sass-and-compass-color-functions.md

```diff
@@ -0,0 +1,28 @@
+# Sass and Compass Color Functions
```

Comment:
Are these all native Sass? Which ones are from Compass? Can we label which ones are from where? Perhaps only show functions from a single source in this TIL? Thoughts?